test/unit/mws/test_feeds_client.rb

```diff
@@ -30,7 +30,7 @@ def test_submitting_binary_file_feed
     }
 
     @client.stub(:run, nil) do
-      random_byte_stream = Random.bytes(32)
+      random_byte_stream = "\xB1\x9E\xFC\x80\xDBK\xCC\xC7F\x95}\v\x99O\xD9\x04\xB4\xE4V\xC9\x81\xF0\xEB\"\xCDJ\n\x9BM\xFA\x16\x95".b
```

Comment:
Just a quick note, it seems `#bytes` does exist in the older rubies as an instance method, so I'll bring it back.